flink-connectors/flink-connector-hive/src/test/java/org/apache/flink/table/catalog/hive/factories/HiveCatalogFactoryTest.java

```diff
@@ -108,6 +109,26 @@ public void testCreateHiveCatalogWithHadoopConfDir() throws IOException {
         assertEquals(mapredVal, ((HiveCatalog) actualCatalog).getHiveConf().get(mapredKey));
     }
 
+    @Test
+    public void testCreateHiveCatalogWithIllegalHadoopConfDir() throws IOException {
+        final String catalogName = "mycatalog";
+        final String illegalHadoopConfDir = " " + tempFolder.newFolder().getAbsolutePath();
```

Comment:
Why we need to add a space character to the conf dir?

Reply:
> Why we need to add a space character to the conf dir?

Set a wrong  hadoop conf path.
Sorry, the rules of the test method are not familiar, I will learn it later, thank you for your guidance.